### PR TITLE
Cp 138 add team member

### PR DESCRIPTION
**Pull request by RihabRJAB**

### CODE REVIEW

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx [hunk 1]
```diff
@@ -1,62 +1,282 @@
...
+      <Grid container spacing={1} className={classes.grid}>
+        <Grid item xs={6}>
+          {mvpInfo.team && teamMembers.length > 0 && (
+            <List className={classes.root}>
+              {teamMembers.map((member) => (
+                <div key={member.id}>
+                  <ListItem alignItems="flex-start">
+                    <ListItemAvatar>
+                      <Avatar
+                        alt="Remy Sharp"
```
Comment (zuming-acn) [inline cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:98]: Who is Remy Sharp ?

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx [hunk 2]
```diff
@@ -1,62 +1,282 @@
...
+                  <Grid item xs={3}>
+                    <Button className={classes.addButton} onClick={closeForm}>
+                      Discard
+                    </Button>
+                  </Grid>
+                  <Grid item xs={1} />
+                  <Grid item xs={3}>
+                    <Button
+                      disabled={
+                        firstName.length === 0 ||
```
Comment (zuming-acn) [inline cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:255]: What about create a function for this logic called field validator ?

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx (hunk 1 above)
Comment (RihabRJAB) [reply cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:98]: none lol

File: cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx (hunk 2 above)
Comment (RihabRJAB) [reply cockpit-frontend/src/components/TeamMemberList/TeamMemberList.jsx:255]: done

**Pull request merged**